networkx/generators/tests/test_directed.py

```diff
@@ -41,6 +41,18 @@ def test_create_using_keyword_arguments(self):
         G = gnc_graph(100, seed=1)
         MG = gnc_graph(100, create_using=MultiDiGraph(), seed=1)
         assert sorted(G.edges()) == sorted(MG.edges())
+        G = gn_graph(1, seed=1)
+        MG = gn_graph(1, create_using=MultiDiGraph(), seed=1)
+        assert G.number_of_nodes() == 1 and G.number_of_edges() == 0
+        assert MG.number_of_nodes() == 1 and MG.number_of_edges() == 0
+        G = gnr_graph(1, 0.5, seed=1)
+        MG = gnr_graph(1, 0.5, create_using=MultiDiGraph(), seed=1)
+        assert G.number_of_nodes() == 1 and G.number_of_edges() == 0
+        assert MG.number_of_nodes() == 1 and MG.number_of_edges() == 0
+        G = gnc_graph(1, seed=1)
+        MG = gnc_graph(1, create_using=MultiDiGraph(), seed=1)
+        assert G.number_of_nodes() == 1 and G.number_of_edges() == 0
+        assert MG.number_of_nodes() == 1 and MG.number_of_edges() == 0
```

Comment:
Just an organizational note, but I'd prefer to split this into it's own test to make it easier for future readers to understand what this is testing. For example, something like:

```python
def test_gn_graph_generators_single_node(self):
   ...
```

Also, this seems like a prime candidate for test parameterization, since this is *really* just a single test case that is run over 3 different functions (`gn_`, `gnr`, and `gnc`) and two different values of `create_using` (None and MultiDiGraph).

You might want to check out the [pytest docs on test parametrization](https://docs.pytest.org/en/6.2.x/parametrize.html): pay special attention to the section on multiple parametrizations!